agenda-services/src/main/java/org/exoplatform/agenda/service/AgendaCalendarServiceImpl.java

```diff
@@ -217,7 +218,7 @@ public Calendar createCalendarInstance(long ownerId) {
   public Calendar createCalendarInstance(long ownerId, long userIdentityId) throws IllegalAccessException {
     boolean canEditCalendar = Utils.canEditCalendar(identityManager, spaceService, ownerId, userIdentityId);
     boolean canCreateEvent = Utils.canCreateEvent(identityManager, spaceService, ownerId, userIdentityId);
-    boolean canInviteeEdit = Utils.canInviteeEdit(identityManager, spaceService, ownerId);
+    boolean canInviteeEdit = Utils.canInviteeEdit(identityManager, spaceService, ownerId, RestUtils.getCurrentUser());
```

Comment:
The toggle for enabling an invitee to edit if he is member of a space that does not have redactors, no need to add the user to this function.
The logic is as follows :
 - If there is at least a redactor in the space, the toggle of invite can Edit will be disabled and can not be enabled
 - If there is no redactor, the toggle is enabled and we can choose to let event invitees to edit the event
 - All managers and redactors in both cases are able to edit events

Reply:
How can I verify that this member is not a redactor if there is redactors in space to decide whether  toggle button will be enabled or disabled?